src/components/SubHeader/SubHeader.js

```diff
@@ -48,15 +48,15 @@ export const SubHeader = ({ user, setCurrentLink }) => {
 
   const subHeaderMenuList = [
     {
-      label: <span className="margin-right-1 text-no-wrap">Resources</span>,
+      label: <span className="margin-right-1 text-no-wrap font-body-sm">Resources</span>,
       items: [
         { menu: "FAQs", link: `/faqs` },
         { menu: "Resources", link: `/resources` },
       ],
     },
     {
       label: (
-        <span className="margin-right-1 text-no-wrap">
+        <span className="margin-right-1 text-no-wrap font-body-sm">
           <span
             className="margin-right-20 menu-item-separator display-none desktop-lg:display-inline-block"
```

Comment:
This is not quite related, but the it doesn't look like this margin class name exists, I think it should be updated to the following:
```suggestion
            className="margin-right-2 menu-item-separator display-none desktop-lg:display-inline-block"
```